### PR TITLE
Convert pull-kops from bash to use kubernetes_kops scenario

### PR DESCRIPTION
**Pull request by krzyzacy**
/assign

### CODE REVIEW

**Comment by krzyzacy** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/46662/pull-kubernetes-e2e-kops-aws-scenario/7/
and
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/kops/2987/pull-kops-e2e-kubernetes-aws-scenario/5/

I'll migrate stuff over :-)

**Comment by krzyzacy** [comment]
The equivalent -scenario runs are green, but also help me check the log see if it is running the right thing.
 
/assign @fejta @zmerlynn @justinsb @chrislovecnm 
cc @kubernetes/kops-reviewers

File: jenkins/bootstrap_test.py [hunk 1]
```diff
@@ -1564,71 +1563,33 @@ def check(job, name):
...
-            if is_modern(name):
-                self.assertGreater(job['timeout'], 0)
+            self.assertNotIn('json', job)
+            self.assertGreater(job['timeout'], 0)
             return job_name
 
         self.check_bootstrap_yaml(
             'job-configs/kubernetes-jenkins-pull/bootstrap-security-pull.yaml',
-            check, use_json=is_modern)
+            check, use_json=True)
```
Comment (fejta) [inline jenkins/bootstrap_test.py]: so exciting!

**Comment by fejta** [comment]
fantastic!
/lgtm

**Comment by krzyzacy** [comment]
cc @ixdy fyi

**Pull request merged**